src/02-common-principles.md

```diff
@@ -187,10 +187,9 @@ levels unless they are overridden by a file at the lower level. For example,
 TR to a specific value. If one of the runs has a different TR than the one
 specified in that file, another `sub-*_task-rest_bold.json` file can be placed
 within that specific series directory specifying the TR for that specific run.
-There is no notion of "unsetting" a key/value pair. For example if there is a
-JSON file corresponding to particular participant/run defining a key/value and
-there is a JSON file on the root level of the dataset that does not define this
-key/value it will not be "unset" for all subjects/runs. Files for a particular
+There is no notion of "unsetting" a key/value pair. Once a key/value pair is 
+set in a given level in the dataset, lower down in the hierarchy that 
+key/value pair will always have some assigned value. Files for a particular 
```

Comment:
While we're here, let's add newlines after sentences, so that any future edits to the sentence can have a reasonably confined diff:

```suggestion
There is no notion of "unsetting" a key/value pair.
Once a key/value pair is set in a given level in the dataset, lower down in
the hierarchy that key/value pair will always have some assigned value.
Files for a particular 
```